core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -173,6 +173,30 @@ public static void merge(Map<?, ?> properties) {
     checkUserFileBufferBytes();
   }
 
+  /**
+   * Merges the current configuration properties with site properties. Some site properties, e.g.
+   * alluxio.logs.dir, alluxio.conf.dir, alluxio.site.conf.dir are ignored.
+   *
+   * @param siteProperties The {@link Properties} to be merged
+     */
+  public static void mergeSiteProperties(Map<?, ?> siteProperties) {
```

Comment:
this duplication is hard to maintain. I suggest either removing the invalid entries from `siteProps` before calling `merge` or adding a `filterIgnoredSiteProperties` argument to `merge`

Reply:
Done.

Reply:
not updated?

Reply:
Haven't pushed yet. Coming soon.